apps/chat/src/app.module.ts

```diff
@@ -0,0 +1,49 @@
+import { Module } from '@nestjs/common';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { MongooseModule, MongooseModuleOptions } from '@nestjs/mongoose';
+import { TypeOrmModule, TypeOrmModuleOptions } from '@nestjs/typeorm';
+import { ChatModule } from './chat/chat.module';
+import mongoConfig from './config/mongo.config';
+import mysqlConfig from './config/mysql.config';
+
+@Module({
+  imports: [
+    ConfigModule.forRoot({
+      isGlobal: true,
+      load: [mysqlConfig, mongoConfig],
+    }),
+    TypeOrmModule.forRootAsync({
+      inject: [ConfigService],
+      useFactory: async (configService: ConfigService) => {
+        const boardTypeOrmModuleOptions: TypeOrmModuleOptions = {
+          type: 'mysql',
+          host: configService.get('mysql.host'),
```

Comment:
```suggestion
        const chatTypeOrmModuleOptions: TypeOrmModuleOptions = {
          type: 'mysql',
          host: configService.get('mysql.host'),
```

TypeORM 모듈의 이름이 chat으로 바뀌어야 할 것 같습니다!

Reply:
넵 명칭 바꾸겠습니다! 감사합니다